src/bp/sdk/botpress.d.ts

```diff
@@ -1217,25 +1259,9 @@ declare module 'botpress/sdk' {
     params?: any
   }
 
-  export interface FlowVariableConfig {
+  export interface FlowVariableConfig extends FormDefinition {
     /** Id use by FlowVariable to refer to this FlowVariableConfig */
-    name: string
-    /** Translation key to be used for display */
-    label: string
-    params: FlowVariableParameter[]
-  }
-
-  export interface FlowVariableParameter {
-    /** Key to place in parms of the FlowVariable */
-    name: string
-    /** Translation key to be used for display */
-    label: string
-    /** Type of control to use to enter the parameter */
-    control: string
-    /** Specific settings for the control. Optional. */
-    controlData?: any
-    /** Indicates if this parameter should be placed in the advanced section */
-    isAdvanced?: boolean
+    type: string
   }
```

Comment:
Please avoid extending definitions. We try to use the type extension instead (so we have a consistent codebase)

`export type FlowVariableConfig = { type: string } & FormDefinition`